docs/csharp/roslyn-sdk/get-started/semantic-analysis.md

```diff
@@ -112,7 +112,7 @@ To finish this tutorial, let's build a LINQ query that creates a sequence of all
 
 [!code-csharp[Access the sequence of members on the string type](../../../../samples/csharp/roslyn-sdk/SemanticQuickStart/Program.cs#9 "Access the sequence of members on the string type.")]
 
-That source sequence contains all members, including properties and fields, so filter it using the <xref:System.Collections.Immutable.ImmutableArray%601.OfType%2A?displayProperty=nameWithType> method to find elements that are <xref:Microsoft.CodeAnalysis.IMethodSymbol?diplayProperty=nameWithType> objects:
+That source sequence conatins all members, including properties and fields, so filter it using the <xref:System.Collections.Immutable.ImmutableArray%601.OfType%2A?displayProperty=nameWithType> method to find elements that are <xref:Microsoft.CodeAnalysis.IMethodSymbol?displayProperty=nameWithType> objects:
```

Comment:
Oops, I knew the "conatins" was going to conflict. I'll re-PR later

Reply:
I think I resolved the conflicts and merged it. Did I mess up the merge?

Reply:
Yeah, I had a separate PR that touched this line. The conflict resolution reverted it since it landed first.
I usually lump changes that happen to the same line, but I felt it was better to batch the  `diplay` stuff.